### ISSUE TITLE
dynamic v-on

### ISSUE THREAD

**Issue by PavelPolyakov** [vuejs/Discussion#253]
Hi,

Assuming we have an element and we want to attach some onclick event, or any other event, which is referring our ```methods``` - how it's possible to do that?

In the documentation there is little regarding this, and we are assuming that vm.$on ( http://vuejs.org/api/instance-methods.html#Events )  is only for the events inside the component, not attached with the DOM.

@yyx990803 , please advice.
@12wrigja, fyi.

Regads,

**Comment by azamat-sharapov** [comment]
If I understand you correctly, you want to attach event listener to element from Model, instead of doing it in template. If so, it's just javascript question and here is demo: http://jsbin.com/tisujohato/edit?html,js,output

**Comment by PavelPolyakov** [comment]
@azamat-sharapov 
Thanks,

I thought something more complex stands behind the v-on.
Do you think if I would remove my component with ```$destroy```, for example, the event listener would gone as well ?

Regards,

**Comment by azamat-sharapov** [comment]
Yeah, it would be gone, you would have to attach event listener again, when you insert it again.
